### PR TITLE
Added EOL Warning to the ChefDK Release Notes

### PR DESCRIPTION
**Pull request by dishanktiwari2501**
## Description

Added EOL Warning to the ChefDK Release Notes as the product is deprecated

## Related PRs

https://github.com/chef/chef-web-docs/issues/3459

### CODE REVIEW

File: content/release_notes_chefdk.md [hunk 1]
```diff
@@ -10,6 +10,8 @@ product = []
 <!-- markdownlint-disable-file -->
 <!-- cSpell:disable  -->
 
+{{% EOL_manage %}}
```
Comment (tas50) [inline content/release_notes_chefdk.md]: This is the wrong EOL warning.

Comment (kagarmoe) [reply content/release_notes_chefdk.md]: Yep, find or write the DK EOL

**Review by kagarmoe — CHANGES_REQUESTED** [review]
Look for the ChefDK EOL--you'll need to write one if it doesn't exist already.

File: themes/docs-new/layouts/shortcodes/EOL_chefdk.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<div class="admonition-warning"><p class="admonition-warning-title">Warning</p><div class="admonition-warning-text">
+
+ChefDK is [deprecated](/versions/#deprecated-products-and-versions) and no
```
Comment (tas50) [inline themes/docs-new/layouts/shortcodes/EOL_chefdk.md:3]: There's no need to contact an account rep. Users should download Chef Workstation, which includes all the functionality of ChefDK and a whole lot more.

**Review by kagarmoe — APPROVED** [review]

**Pull request merged**